pkg_security.es.Rmd

```diff
@@ -1,83 +1,83 @@
   
-  - No imprimas la clave de la API ni siquiera en modo verboso en ningún mensaje, advertencia o error.
+  - No imprimas la clave de la API en ningún mensaje, advertencia o error, ni siquiera en modo verboso.
 
-- En la plantilla de emisión de GitHub, debe indicarse que no se comparta ninguna credencial. Si un usuario de tu paquete comparte accidentalmente las credenciales en una incidencia, asegúrate de que es consciente de ello para que pueda revocar la clave (es decir, pregúntale explícitamente en una respuesta si se ha dado cuenta de que ha compartido su clave).
+  - La plantilla de issues en GitHub debe indicar no compartir ninguna credencial. Si alguien comparte accidentalmente las credenciales en un issue, asegúrate de avisarle para que revoque la clave (es decir, pregúntale explícitamente si se ha dado cuenta de que ha compartido su clave).
 
-### Secretos en paquetes y desarrollo
+### Credenciales en paquetes y desarrollo
 
-Tendrás que proteger tus secretos igual que proteges los secretos de los usuarios, pero hay más cosas que hay que tener en cuenta y que hay que tener en cuenta.
+Durante el desarrollo del paquete tendrás que proteger tus credenciales igual que proteges las credenciales de quienes usan tu paquete, pero hay más cosas a considerar y tener en cuenta.
 
-#### Secretos y peticiones registradas en las pruebas
+#### Credenciales y consultas guardadas en los tests
 
-Si utilizas [`vcr`](https://docs.ropensci.org/vcr/) o [`httptest`](https://enpiar.com/r/httptest/) en las pruebas para almacenar en caché las respuestas de la API, tienes que asegurarte de que las peticiones grabadas / los accesorios no contienen secretos. Consulta la [`vcr` guía de seguridad](https://books.ropensci.org/http-testing/security-chapter.html) y [`httptest` orientación "Redactar y modificar las solicitudes grabadas"](https://enpiar.com/r/httptest/articles/redacting.html)e inspecciona tus solicitudes/accesorios grabados antes de enviarlos por primera vez para asegurarte de que tienes la configuración correcta.
+Si utilizas [`vcr`](https://docs.ropensci.org/vcr/) o [`httptest`](https://enpiar.com/r/httptest/) en los test para almacenar las respuestas de la API en caché, tienes que asegurarte de que las consultas grabadas o fixtures no contengan credenciales. Consulta la [guía de seguridad de `vcr`](https://books.ropensci.org/http-testing/security-chapter.html) y [la guía "Redacting and Modifying Recorded Requests" de `httptest`](https://enpiar.com/r/httptest/articles/redacting.html) e inspecciona tus consultas grabadas y fixtures antes de hacer un primer commit para asegurarte de que tienes la configuración correcta.
```

Comment:
La palabra consulta aqui me hace un poco de ruido.  Es consulta que viene de request? (como petición en una API) o de query (como consulta en una base de datos)?

Reply:
La palabra fixtures podría traducirse como funcionalidad/funcionalidades ?

Reply:
La mayoría de los otros títulos de recursos están en español: Redacting and Modifying Recorded Requests.  Podría ser: _Redactar y modificar las solicitudes/consultas/peticiones registradas_

Reply:
> . Es consulta que viene de request? (como petición en una API) o de query (como consulta en una base de datos)?

Viene de request a la API. Creo que la idea es que hacés una consulta a la API y grabás la respuesta. Entonces cuando hacés los tests, las funciones que hacen la consulta, reciben la respuesta grabada en vez de mandar la consulta a la API de verdad. 


> La palabra fixtures podría traducirse como funcionalidad/funcionalidades ?

No, la documentación de httptest llama "fixtures" a las requests guardadas en cache. No encontramos una buena forma de traducirlo al español y como es medio una palabra técnica decidimos que podríamos dejarla. Ahora se me ocurre que "artefactos" podría estar cerca. 

> La mayoría de los otros títulos de recursos están en español: Redacting and Modifying Recorded Requests. Podría ser: Redactar y modificar las solicitudes/consultas/peticiones registradas


Con Pao consideramos que si linkeamos a un recurso que está en inglés, no tiene sentido traducir el título. Hay muchos recursos donde el texto del link está en español pero es cuando lo describimos (por ejemplo. `[capítulo de seguridad del libro HTTP testing in R](https://books.ropensci.org/http-testing/security-chapter.html)`

Reply:
Perfecto todo. Está para hacer un merge entonces.